src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -364,7 +364,7 @@ private async Task RestartWorkerChannel(string runtime, string workerId)
             }
             else if (_jobHostLanguageWorkerChannelManager.GetChannels().Count() == 0)
             {
-                _logger.LogError("Exceeded language worker restart retry count for runtime:{runtime}. Shutting down Functions Host", runtime);
+                _logger.LogError("Exceeded language worker restart retry count for runtime:{runtime}. Proactively recycling the Functions Host to recover", runtime);
```

Comment:
Can we keep the 'Shutting down Functions Host' and append ` Proactively recycling the Functions Host to recover`?

Reply:
I like the suggestion from Pragna, As long as we're showing that it will be back up soon.